### ISSUE TITLE
Test uploading a sheet and downloading a KMP

### ISSUE THREAD

**Issue by jienius** [eddieantonio/predictive-text-studio#32]

**Comment by eddieantonio** [comment]
**Blocked by** #40

**Comment by eddieantonio** [comment]
This _might_ be a duplicate of #27.

**Comment by eddieantonio** [comment]
Fixed by #110
